app/data_model/database.py

```diff
@@ -68,15 +68,17 @@ def __repr__(self):
 
 class Database:
 
-    def __init__(self, database_url, database_setup_retry_count, database_setup_retry_delay):
+    def __init__(self, database_driver, database_url, username, password, database_setup_retry_count, database_setup_retry_delay):
```

Comment:
We're calling two different things `database_url` here. We should rename `database_url` (AWS calls this an endpoint in RDS) or separate into host and port.